android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -14,8 +14,8 @@
 		"startBias": ["Avoid [Tundra]"],
 		"preferredVictoryType": "Scientific",
 
-		"startIntroPart1": "",
-		"startIntroPart2": "",
+		"startIntroPart1": "May the gods bless you, great Nebuchadnezzar, ruler of ancient Babylon! Built 5000 years ago, Babylon emerged as an empire in 1800 BC only, thanks to godlike Hammurabi, The Giver of Law. Fallen under the Kassites assaults then the Assyrian domination, Babylon reborned from its ashes, gaining back its independance, and became the beacon for Arts and Learning in the ancient world. O Nebuchadnezzar, after your death, Babylon sadly didn't last long, conquered by the mighty Persians, and then by the Greeks, until the great city disappeared forever in 141 BC.",
```

Comment:
Remove 'only'
Fallen -> Falling
Reborned -> was reborn

Instead of 'sadly didn't last long' put 'quickly fell'